rules/avoid-injections.js

```diff
@@ -26,6 +26,8 @@ function check(context, node) {
   const statement = node.callee.property.name;
   const queryNode = node.arguments[0];
 
+  if (queryNode === undefined) return;
```

Comment:
I would just make this part of the early return below.

Reply:
update in https://github.com/AntonNiklasson/eslint-plugin-knex/pull/6/commits/add9101aec13ea040fd67548daeeff1c52e34910